server.lua

```diff
@@ -77,11 +77,11 @@ function GetGuildId (guildName)
   return result
 end
 
-function DiscordRequest(method, endpoint, jsondata)
+function DiscordRequest(method, endpoint, jsondata, reason)

```

Comment:
Won't changing this require all function calls for it requiring to pass the `reason` ?

Reply:
If you don't pass the `reason` parameter, it will be nil, and the Discord API won't add a reason for that action in the audit log.

Reply:
If every request can have a reason added to it, should all the methods have it as an optional parameter maybe?

Reply:
Considering that GET requests don't get logged by Discord anyways, I feel like this solution to logging requests is sufficient enough.